docker/Dockerfile.fedora-base

```diff
@@ -9,11 +9,15 @@ RUN dnf install -y \
     automake \
     cpio \
     elfutils-devel \
+    elfutils-libs \
     gcc-c++ \
     libtool \
+    libxml2-libs \
     libxml2-devel \
     python3-koji \
     python3-mock \
     python3-pyxdg \
     six \
+    file \
```

Comment:
I don't mean to be pedantic but can we put these just in the testing container not just the base?

Reply:
Not pedantic at all - that's totally reasonable to do.